pkg/lib/verifier/rule_builder_util.go

```diff
@@ -20,159 +20,73 @@ import (
 
 //getPcrMatchesConstantRules method will create PcrMatchesConstantRule and return the rule
 //return nil if error occurs
-func getPcrMatchesConstantRules(pcrs []types.PcrIndex, flavor *hvs.Flavor, pcrLogData *types.PCRS, marker common.FlavorPart) ([]rules.Rule, error) {
+func getPcrMatchesConstantRules(pcrLogData *types.PCRS, marker common.FlavorPart) ([]rules.Rule, error) {
```

Comment:
Are these utility functions still needed?  They were intended to automate the creation of multiple rules based on flavor.

Reply:
These methods will be called and create the rule for verification based on the rule name mentioned in flavor part json.